app/views/templates/_form.html.erb

```diff
@@ -12,6 +12,8 @@
     <li class="active"><a id="primary_tab" href="#primary" data-toggle="tab"><%= _("Template") %></a></li>
     <li><a href="#template_inputs" data-toggle="tab"><%= _("Inputs") %></a></li>
     <%= render "#{@type_name_plural}/custom_tab_headers" unless type == 'ptable'%>
+    <%= render_pagelets_for(:job_template_tab_headers) unless type == 'ptable' %>
```

Comment:
I have a feeling this is not only used for job templates but also for provisioning templates and report templates (partition tables are ruled out by the condition)

Reply:
You are right, it is also used for provisioning_templates, partition_tables (I have a condition for it too, I will delete it), and report_templates.
Sorry for the confusion, I'm working on the fix.